Sources/TidepoolKit/Internal/View Controllers/LoginSignupNavigationController.swift

```diff
@@ -26,9 +26,9 @@ class LoginSignupNavigationController: UINavigationController, TLoginSignup {
 
         super.init(navigationBarClass: UINavigationBar.self, toolbarClass: UIToolbar.self)
 
-        navigationBar.isTranslucent = false
-        navigationBar.barTintColor = .secondarySystemBackground
-        navigationBar.shadowImage = UIColor.clear.image()
+//        navigationBar.isTranslucent = false
+//        navigationBar.barTintColor = .secondarySystemBackground
+//        navigationBar.shadowImage = UIColor.clear.image()
```

Comment:
Needed? Can these be removed?

Reply:
I wasn't sure if we'd need to restore them. I commented them out to fix an issue in DIY.  But I just tested in Tidepool Loop, and it looks fine, so I'll remove. Thanks!